src/css/custom.css

```diff
@@ -141,6 +173,13 @@ html[data-theme='dark']::before {
 html {
   font-size: var(--ifm-font-size-base);
   letter-spacing: -0.01em;
+  font-family: var(--ifm-font-family-base);
+}
+
+body {
+  font-family: var(--ifm-font-family-base);
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 }
 
```

Comment:
We should change the font in the Algolia search box as well
```
[data-theme='dark'] .DocSearch,
[data-theme='light'] .DocSearch {
  font-family: 'Inter', -apple-system, sans-serif;
}
```

Reply:
nice catch